src/shared/enums/entityType.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum entityType {
```

Comment:
same as previous, I think the word `entity` could be renamed to something more descriptive, like `post` for example.

Reply:
entity here refers to (post | optionsGroup | option) entities. So what would you suggest to name it.